Build fails with "Failed to run task: 'npm install' failed"

```
2021-10-18T07:24:54.8492939Z [INFO] npm WARN The package @testing-library/react is included as both a dev and production dependency.
2021-10-18T07:24:54.8495571Z [INFO] npm WARN The package @types/react is included as both a dev and production dependency.
2021-10-18T07:24:54.8499389Z [INFO] npm WARN The package @types/react-dom is included as both a dev and production dependency.
2021-10-18T07:24:54.8500653Z [INFO] npm WARN The package eslint-loader is included as both a dev and production dependency.
2021-10-18T07:24:54.8501805Z [INFO] npm WARN The package eslint-plugin-import is included as both a dev and production dependency.
2021-10-18T07:24:54.8503061Z [INFO] npm WARN The package eslint-plugin-jsx-a11y is included as both a dev and production dependency.
2021-10-18T07:24:54.8504529Z [INFO] npm WARN The package eslint-plugin-react is included as both a dev and production dependency.
2021-10-18T07:24:54.8505802Z [INFO] npm WARN The package eslint-plugin-react-hooks is included as both a dev and production dependency.
2021-10-18T07:24:54.8506535Z [INFO] 
2021-10-18T07:24:54.8568230Z [INFO] npm ERR! code EINTEGRITY
2021-10-18T07:24:54.8570675Z [INFO] npm ERR! Verification failed while extracting react-tutorial@https://github.com/deuex-solutions/react-tutorial/tarball/master:
2021-10-18T07:24:54.8572574Z [INFO] npm ERR! Verification failed while extracting react-tutorial@https://github.com/deuex-solutions/react-tutorial/tarball/master:
2021-10-18T07:24:54.8577445Z [INFO] npm ERR! sha512-zuCdUOXiUstfSE1cZq4+yk8/jpkjKUholNRd2bhw55SxoF/32yNUtU9JbiLxvkCUcdiiggZJwuW7Evpljmu0AQ== integrity checksum failed when using sha512: wanted sha512-zuCdUOXiUstfSE1cZq4+yk8/jpkjKUholNRd2bhw55SxoF/32yNUtU9JbiLxvkCUcdiiggZJwuW7Evpljmu0AQ== but got sha512-PVHe43HCsmOsCdLIFBtW3tp7xlh2JmaOH871dXjMkycWTSvy8LW10OwkvEL2BzDvn8SERwdTblHHcBvzeg8Hwg==. (142659 bytes)
2021-10-18T07:24:54.9934570Z [INFO] 
2021-10-18T07:24:54.9935104Z [INFO] npm ERR! A complete log of this run can be found in:
2021-10-18T07:24:54.9936272Z [INFO] npm ERR!     /home/runner/.npm/_logs/2021-10-18T07_24_54_879Z-debug.log
2021-10-18T07:24:55.0117842Z [INFO] ------------------------------------------------------------------------
2021-10-18T07:24:55.0119176Z [INFO] Reactor Summary for catalog 0.6.0-SNAPSHOT:
2021-10-18T07:24:55.0119784Z [INFO] 
2021-10-18T07:24:55.0120824Z [INFO] catalog ............................................ SUCCESS [  0.236 s]
2021-10-18T07:24:55.0121502Z [INFO] common ............................................. SUCCESS [ 14.486 s]
2021-10-18T07:24:55.0122509Z [INFO] catalog-rest-service ............................... FAILURE [03:59 min]
2021-10-18T07:24:55.0123243Z [INFO] Catalog Distribution ............................... SKIPPED
2021-10-18T07:24:55.0124104Z [INFO] ------------------------------------------------------------------------
2021-10-18T07:24:55.0124738Z [INFO] BUILD FAILURE
2021-10-18T07:24:55.0125501Z [INFO] ------------------------------------------------------------------------
2021-10-18T07:24:55.0126134Z [INFO] Total time:  04:20 min
2021-10-18T07:24:55.0126791Z [INFO] Finished at: 2021-10-18T07:24:55Z
2021-10-18T07:24:55.0127589Z [INFO] ------------------------------------------------------------------------
2021-10-18T07:24:55.0131319Z [ERROR] Failed to execute goal com.github.eirslett:frontend-maven-plugin:1.11.0:npm (npm install) on project catalog-rest-service: Failed to run task: 'npm install' failed. org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1) -> [Help 1]
2021-10-18T07:24:55.0169664Z [ERROR] 
2021-10-18T07:24:55.0171747Z [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
2021-10-18T07:24:55.0173367Z [ERROR] Re-run Maven using the -X switch to enable full debug logging.
2021-10-18T07:24:55.0174394Z [ERROR] 
2021-10-18T07:24:55.0199068Z [ERROR] For more information about the errors and possible solutions, please read the following articles:
2021-10-18T07:24:55.0202432Z [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
2021-10-18T07:24:55.0203449Z [ERROR] 
2021-10-18T07:24:55.0203964Z [ERROR] After correcting the problems, you can resume the build with the command
2021-10-18T07:24:55.0204940Z [ERROR]   mvn <args> -rf :catalog-rest-service
2021-10-18T07:24:55.0505493Z ##[error]Process completed with exit code 1.
```

cc: @darth-coder00

--- a/catalog-rest-service/src/main/resources/ui/package.json
+++ b/catalog-rest-service/src/main/resources/ui/package.json
@@ -81,8 +81,7 @@
     "stream-http": "^3.2.0",
     "styled-components": "^5.2.3",
     "tailwindcss": "^2.1.4",
-    "to-arraybuffer": "^1.0.1",
-    "react-tutorial": "https://github.com/deuex-solutions/react-tutorial/tarball/master"
+    "to-arraybuffer": "^1.0.1"
   },
   "scripts": {
     "start": "NODE_ENV=development BABEL_ENV=development webpack serve --config ./webpack.config.dev.js --env development",

--- a/catalog-rest-service/src/main/resources/ui/src/components/tour/Tour.tsx
+++ b/catalog-rest-service/src/main/resources/ui/src/components/tour/Tour.tsx
@@ -1,11 +1,11 @@
 import { AxiosResponse } from 'axios';
 import { observer } from 'mobx-react';
 import React, { useEffect, useState } from 'react';
-import ReactTutorial from 'react-tutorial';
+// import ReactTutorial from 'react-tutorial';
 import { searchData } from '../../axiosAPIs/miscAPI';
 import { PAGE_SIZE } from '../../constants/constants';
 import { SearchIndex } from '../../enums/search.enum';
-import { useTour } from '../../hooks/useTour';
+// import { useTour } from '../../hooks/useTour';
 
 type Steps = {
   content: string;
@@ -73,8 +73,8 @@ const getSteps = (value: string) => {
 };
 
 const Tour = () => {
-  const { isTourOpen, handleIsTourOpen } = useTour();
-  const [steps, setSteps] = useState<Steps[]>([]);
+  // const { isTourOpen, handleIsTourOpen } = useTour();
+  const [, setSteps] = useState<Steps[]>([]);
 
   useEffect(() => {
     searchData('', 1, PAGE_SIZE, '', '', '', SearchIndex.TABLE).then(
@@ -87,7 +87,7 @@ const Tour = () => {
 
   return (
     <div>
-      {isTourOpen ? (
+      {/* {isTourOpen ? (
         <ReactTutorial
           disableKeyboardNavigation
           showNumber
@@ -98,7 +98,7 @@ const Tour = () => {
           steps={steps}
           onRequestClose={() => handleIsTourOpen(false)}
         />
-      ) : null}
+      ) : null} */}
     </div>
   );
 };

